.github/workflows/ci-release.yml

```diff
@@ -70,23 +70,33 @@ jobs:
         submodules: true
 
     # Install GNAT, we only need a system compiler for Ubuntu ARM until we have
-    # Alire-indexed releases for it.
-
-    - name: Install Alire toolchain
-      if: matrix.platform.id != 'aarch64-linux'
-      uses: alire-project/alr-install@v2
-      with:
-        crates: gnat_native gprbuild
-        prefix: alire_prefix
+    # Alire for ARM64/Linux (on Ubuntu 22.04) release.
 
     - name: Install system toolchain (Ubuntu ARM)
       if: matrix.platform.id == 'aarch64-linux'
       run: |
         sudo apt-get install -y gnat gprbuild
         echo ALIRE_TESTSUITE_DISABLE_DOCKER=true >> $GITHUB_ENV
-        echo USE_EXTERNAL=use_external >> $GITHUB_ENV
-      # GNAT 10 has a bug that fails in the testsuite. Also, we need to disable
-      # Docker tests on Ubuntu ARM.
+        # We need to disable Docker tests on Ubuntu ARM.
+
+    - name: Bootstrap Alire (Ubuntu ARM)
+      if: matrix.platform.id == 'aarch64-linux'
+      uses: alire-project/setup-alire@v5
+      with:
+        branch: "master" # build from source
```

Comment:
Why is an `alr` being installed here? I'm failing to see the need.

Reply:
Otherwise `alire-project/alr-install@v2` will install alr `2.1.0` release build on Ubuntu 24.04 and it fails on Ubuntu 22.04 :disappointed:

Reply:
I see, understood. And this can be removed once we have 2.1.1 for Ubuntu 22.04 aarch64.